### ISSUE TITLE
Jira title no longer displaying when clicking "Start timer"

### ISSUE THREAD

**Issue by VagishVela** [toggl/track-extension#2085]
* **OS version**: Max OS 12.1
* **Browser version**: Firefox Developer Edition 101.0b2 (64-bit)
* **Extension version**: 2.0.8

#### Relevant integration (if any):

https://*.atlassian.net/

### 🐛 Describe the bug

Jira title no longer displaying when clicking "Start timer", this is happening on the Jira board.
<img width="446" alt="Screen Shot 2022-05-10 at 7 33 09 AM" src="https://user-images.githubusercontent.com/7334882/167563602-371e1bde-8094-47ed-8b7f-d1e2e6fee35e.png">

### Expected behaviour

It should include the title along with the ticket number in the toggl description.

### Steps to reproduce

Steps to reproduce the behaviour:
1. Go to your JIRA sprint board (for a current sprint)
2. Click any ticket
3. Click "Start timer"

### Other details or context
So basically the problem is, the below isn't finding the text as the button is above the h1 now. But I don't know where the issue list pages are or I could fix it.
```
  // Title/summary of the issue - we use the hidden "edit" button that's there for a11y
  // in order to avoid picking up actual page title in the case of issue-list-pages.
  const titleElement = document.querySelector('h1 ~ button[aria-label]');
```

**Comment by markoph** [comment]
Something fishy happened in last update. It is exactly opposite issue I have with GitLab (picking titles only) https://github.com/toggl/track-extension/issues/2083.

**Comment by nicktoggl** [comment]
[Case reference](https://app.intercom.com/a/apps/ayixs927/inbox/inbox/2097915/conversations/106481309740941)

**Comment by nidarasheed** [comment]
Hi, any movement on this? The user followed up with us.

**Comment by KonradDa** [comment]
Hi there, 
is there any progress on this. the bug ist quite annoing

**Comment by VagishVela** [comment]
@KonradDa I don't think so. I'm still having the issue. I would try but I don't understand some of the terminology to fix it and frankly not wanting to spend a couple hours learning it.

**Comment by RALGIE** [comment]
Same here, can you please fix this issue

**Comment by TheSoloJake** [comment]
As per @nunofmn closing this issue as it should be fixed through https://github.com/toggl/track-extension/commit/66816c669d9eda74e9ae5e0c36c2586f55fcab5f
